helm/cluster-vsphere/templates/ipam/assign-ip-pre-install-job.yaml

```diff
@@ -143,11 +143,11 @@ spec:
                 count=$((count+1))
               done
 
-              # patch the cloud-provider-vsphere-helmrelease
+              # patch the kube-vip-cloud-provider helmrelease
               {{- if .Values.global.connectivity.network.loadBalancers.cidrBlocks }}
               ip_list="${ip_list},{{ join "," .Values.global.connectivity.network.loadBalancers.cidrBlocks }}"
               {{- end }}
-              kubectl patch helmrelease -n {{ $.Release.Namespace }} {{ include "resource.default.name" $ }}-cloud-provider-vsphere --type=merge -p '{"spec":{"suspend":false,"values":{"kube-vip-cloud-provider": {"cidrGlobal": "'${ip_list}'"}}}}'
+              kubectl patch helmrelease -n {{ $.Release.Namespace }} {{ include "resource.default.name" $ }}-kube-vip-cloud-provider --type=merge -p '{"spec":{"suspend":false,"values":{"cm":{"data":{"cidr-global":"'${ip_list}'"}}}}}'
```

Comment:
we just need to patch the kube-vip CPI helmrelease now

Reply:
There was a ` ` (space) after `cidr-global:`.
Is that alright?

Reply:
yeah, whitespace in JSON between keys is for readability - it gets interpreted the same either way when it gets parsed. whitespace only matters inside values. I pinched this from cluster-proxmox which works fine.